specification/storagecache/resource-manager/Microsoft.StorageCache/preview/2020-08-13-aml-preview/examples/amlFilesystems_Update.json

```diff
@@ -0,0 +1,89 @@
+{
+  "parameters": {
+    "resourceGroupName": "scgroup",
+    "cacheName": "fs1",
+    "subscriptionId": "00000000-0000-0000-0000-000000000000",
+    "api-version": "2020-07-23-preview",
+    "amlFilesystem": {
+      "tags": {
+        "Dept": "ContosoAds"
+      },
+      "location": "westus",
+      "properties": {
+        "storageCapacityTiB": 3072,
```

Comment:
Most of the properties in this example may not be changed. Is it the convention to include such things in the example?

Reply:
Let me know if any of these can be modified.   Swagger isn't good with create vs modify documentation.  There's readOnly true/false and thats it.  So if create needs it, its also documented for update as well.  Unless you make a new object definition for each operation which causes a lot of definitions.

"storageCapacityTiB"
"zones"
"subnet"
"mountCommand" - Made this readOnly.
"MGSAddress" - Made this readOnly.
"encryptionSettings"
"mainenanceWindow"
"hsmSettings"

Reply:
also readOnly: storageCapacityTiB, zones.

Reply:
ok, Makred it readOnly.
Remove from parameter section of CreateOrUpdate and Update examples.